<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_filter</class>
 <widget class="QDialog" name="Dialog_filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="design_filter_pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>570</y>
     <width>94</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>生成滤波器</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="up_dB_lineEdit">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>505</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="down_dB_lineEdit">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>532</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="down_weight_label">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>532</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>阻带衰减</string>
   </property>
  </widget>
  <widget class="QLabel" name="dB1_label">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>505</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>dB</string>
   </property>
  </widget>
  <widget class="QLabel" name="up_weight_label">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>505</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>通道波动</string>
   </property>
  </widget>
  <widget class="QLabel" name="dB2_label">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>532</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>dB</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edge2_lineEdit">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>532</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="down_edge_label">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>532</y>
     <width>68</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>边缘频率2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edge1_lineEdit">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>505</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="up_edge_label">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>505</y>
     <width>68</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>边缘频率1</string>
   </property>
  </widget>
  <widget class="QComboBox" name="edge2_Hz_Box">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>532</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Hz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KHz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MHz</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="edge1_Hz_Box">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>505</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Hz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KHz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MHz</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="design_label">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>468</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>设计方法</string>
   </property>
  </widget>
  <widget class="QComboBox" name="design_Box">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>468</y>
     <width>87</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>fir</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>iir</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="response_label">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>441</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>响应类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="respone_Box">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>441</y>
     <width>87</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>低通滤波器</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>高通滤波器</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
